app/src/main/analytics.js

```diff
@@ -1,18 +1,20 @@
 import firstRun from 'first-run';
 import Insight from 'insight';
+import {parse} from 'semver';
 import pkg from '../../package';
 import {get as getSetting} from '../common/settings-manager';
 
 const trackingCode = 'UA-84705099-2';
 const insight = new Insight({trackingCode, pkg});
+const version = parse(pkg.version, {loose: true});
```

Comment:
No need to use the `loose` option. We follow semver.

Reply:
Oh, I see. Thanks again for the feedback. 🙌